appveyor.yml

```diff
@@ -0,0 +1,189 @@
+# with the PYTHON_ARCH variable (which is used by CMD_IN_ENV).
+platform:
+    - x64
+
+# all our python builds have to happen in tests_script...
+build: false
+
+init:
+  - cmd: "ECHO %PYTHON_VERSION% PYTEST=%USE_PYTEST% %CONDA_INSTALL_LOCN%"
+
+install:
+  - cmd: set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts;%PATH%;
+  - cmd: set PYTHONUNBUFFERED=1
+  # for obvci_appveyor_python_build_env.cmd
+  - cmd: conda install -c pelson/channel/development --yes --quiet obvious-ci
+  # for msinttypes and newer stuff
+  - cmd: conda config --add channels conda-forge
+  - cmd: conda config --set show_channel_urls yes
+  - cmd: conda config --set always_yes true
+  # For building conda packages
+  - cmd: conda install --yes conda-build jinja2 anaconda-client
+  # this is now the downloaded conda...
+  - conda info -a
+
+  # Fix the appveyor build environment to work with conda build
+  # workaround for missing vcvars64.bat in py34 64bit
+  - cmd: copy ci\appveyor\vcvars64.bat "C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\bin\amd64"
+
+  # For building, use a new environment which only includes the requirements for mpl
+  # same things as the requirements in ci/conda_recipe/meta.yaml
+  # if conda-forge gets a new pyqt, it might be nice to install it as well to have more backends
+  # https://github.com/conda-forge/conda-forge.github.io/issues/157#issuecomment-223536381
+  - conda create -q -n test-environment python=%PYTHON_VERSION%
+    pip setuptools numpy python-dateutil freetype=2.6 msinttypes "tk=8.5"
+    pyparsing pytz tornado "libpng>=1.6.21,<1.7" "zlib=1.2" "cycler>=0.10"
+    nose mock sphinx
+  - activate test-environment
+  - cmd: echo %PYTHON_VERSION% %TARGET_ARCH%
+  - cmd: IF %PYTHON_VERSION% == 2.7 conda install -q functools32
+  # pytest-cov>=2.3.1 due to https://github.com/pytest-dev/pytest-cov/issues/124
+  - if x%USE_PYTEST% == xyes conda install -q pytest "pytest-cov>=2.3.1" pytest-timeout #pytest-xdist
+
+  # Let the install prefer the static builds of the libs
+  - set LIBRARY_LIB=%CONDA_PREFIX%\Library\lib
+  - cmd: 'mkdir lib || cmd /c "exit /b 0"'
+  - copy /Y %LIBRARY_LIB%\zlibstatic.lib lib\z.lib
+  - copy /Y %LIBRARY_LIB%\libpng_static.lib lib\png.lib
+  # These z.lib / png.lib are not static versions but files which end up as
+  # dependencies to the dll file. This is fine for the conda build, but not here
+  # and for the wheels
+  - del %LIBRARY_LIB%\png.lib
+  - del %LIBRARY_LIB%\z.lib
+  - set MPLBASEDIRLIST=%CONDA_PREFIX%\Library\;.
+  # enables the local freetype build
+  - copy ci\travis\setup.cfg .
+  # Show the installed packages + versions
+  - conda list
+
+test_script:
+  # Now build the thing..
+  - '%CMD_IN_ENV% python setup.py develop'
+  # these should show no z, png, or freetype dll...
+  - set "DUMPBIN=%VS140COMNTOOLS%\..\..\VC\bin\dumpbin.exe"
+  #- cmd: '"%DUMPBIN%" /DEPENDENTS lib\matplotlib\_png*.pyd'
+  #- cmd: '"%DUMPBIN%" /DEPENDENTS lib\matplotlib\ft2font*.pyd'
```

Comment:
You might want to uncomment these two to see that they don't include wrong stuff (the next three lines should find prevent that, but to be sure...)